src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -444,8 +444,9 @@ internal enum WellKnownMember
 
         System_IAsyncDisposable__DisposeAsync,
         System_Collections_Generic_IAsyncEnumerable_T__GetAsyncEnumerator,
-        System_Collections_Generic_IAsyncEnumerator_T__WaitForNextAsync,
-        System_Collections_Generic_IAsyncEnumerator_T__TryGetNext,
+        System_Collections_Generic_IAsyncEnumerator_T__MoveNextAsync,
+        System_Collections_Generic_IAsyncEnumerator_T__Current,
```

Comment:
>System_Collections_Generic_IAsyncEnumerator_T__Current [](start = 8, length = 54)

This member does not appear to be actually used in any nontrivial way.  We generally don't verify the existence of the property when verifying the presence of the property accessor.  I recommend removing it unless there is a specific reason this needs to be different.